Metrics/AppMetrics.php

```diff
@@ -23,6 +24,11 @@ class AppMetrics implements PreRequestMetricsCollectorInterface, RequestMetricsC
 
     private float $startedAt = 0;
 
+    /**
+     * @var non-empty-list<float>
+     */
+    private array $buckets = [0.005, 0.01, 0.025, 0.05, 0.075, 0.1, 0.25, 0.5, 0.75, 1.0, 2.5, 5.0, 7.5, 10.0];
```

Comment:
ditto (https://github.com/PromPHP/prometheus_client_php/blob/a283aea8269287dc35313a0055480d950c59ac1f/src/Prometheus/Histogram.php#L64)

Reply:
Without a ctor (see my thought on this here https://github.com/artprima/prometheus-metrics-bundle/pull/116/files#r2062640663) it's impossible to call this method at property level here.